config/service_providers.yml

```diff
@@ -317,3 +317,28 @@ production:
     agency: 'DHS'
     logo: 'cbp.png'
     allow_on_prod_chef_env: 'true'
+
+  # RRB
+  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:RRB:BOS-Pre-Prod':
+    friendly_name: 'Railroad Retirement Board'
+    agency: 'RRB'
+    logo: 'rrb.svg'
+    acs_url: 'https://onlinetest.rrb.gov/login/login/callback'
+    assertion_consumer_logout_service_url: 'https://onlinetest.rrb.gov/login/login/logout'
+    sp_initiated_login_url: 'https://onlinetest.rrb.gov/login/login/Auth'
+    return_to_sp_url: 'https://onlinetest.rrb.gov/login/login/callback'
```

Comment:
Good point! It depends on how their app is configured, I guess. If these are the values RRB gave to Tom, and they tested them in INT already, it should be fine?